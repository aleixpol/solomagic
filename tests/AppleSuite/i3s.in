\ref something0
\mt somethingsomething ina somethingsomething

\ref something1
\mt somethingsomething hina somethingsomething

