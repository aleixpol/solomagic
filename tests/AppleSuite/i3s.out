\ref something0
\mt somethingsomething ina somethingsomething
\ma somethingsomething ingna somethingsomething

\ref something1
\mt somethingsomething hina somethingsomething
\ma somethingsomething hina somethingsomething
